doc/whats-new.rst

```diff
@@ -62,6 +62,11 @@ Breaking changes
   pre-existing array values. This is a safer default than the prior ``mode="a"``,
   and allows for higher performance writes (:pull:`5252`).
   By `Stephan Hoyer <https://github.com/shoyer>`_.
+- The ``__repr__`` of a :py:class:`xarray.Dataset`'s ``attrs``, ``coords``,
```

Comment:
note that `.attrs` is a standard python dict, so there's no custom `repr`:
```suggestion
- The ``__repr__`` of a :py:class:`xarray.Dataset`'s ``coords``
```

Reply:
Actually, there is a `attrs_repr` in `xarray/core/formatting.py`.
But I already changed the `whats_new` entry, probably I'll have to revert that again.